org.eclipse.dawnsci.analysis.api/src/org/eclipse/dawnsci/analysis/api/tree/TreeUtils.java

```diff
@@ -20,11 +20,17 @@
 import java.util.Queue;
 import java.util.Set;
 
+import org.eclipse.january.DatasetException;
 import org.eclipse.january.IMonitor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 
 public class TreeUtils {
 
+	
+	private final static Logger logger = LoggerFactory.getLogger(TreeUtils.class);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)